linalg/operator.hpp

```diff
@@ -343,6 +346,24 @@ class IdentityOperator : public Operator
 };
 
 
+/// Scaled Operator B: x -> a A(x).
+class ScaledOperator : public Operator
+{
+private:
+   const Operator &A_;
+   double a_;
+
+public:
+   /// Create a scalar product operator related to A.
```

Comment:
I don't think "scalar product" is the right wording here. Maybe "Create an operator which is a scalar multiple of A."

Reply:
That is better, thanks.